typings.d.ts

```diff
@@ -22,8 +22,8 @@ declare namespace betterAjvErrors {
     (
       schema: any,
       data: any,
-      errors: ErrorObject[],
-      options: betterAjvErrors.IInputOptions
+      errors: ErrorObject[] | null | undefined,
```

Comment:
1. `ajv.errors` / `validate.errors` have this type.
2. [Your code has a fallback for falsy errors](https://github.com/atlassian/better-ajv-errors/blob/a4a52574b238e9e38b03313178e5b8daae3db432/src/helpers.js#L124), so this is safe.

Reply:
Same here.